lib/slack/web/api/endpoints/chat.rb

```diff
@@ -221,7 +223,7 @@ def chat_postMessage(options = {})
           #   Pass false to disable unfurling of media content.
           # @see https://api.slack.com/methods/chat.scheduleMessage
           # @see https://github.com/slack-ruby/slack-api-ref/blob/master/methods/chat/chat.scheduleMessage.json
-          def chat_scheduleMessage(options = {})
+          def chat_scheduleMessage(options = {}) # rubocop:disable Naming/MethodName
```

Comment:
These are auto-generated files, so this gets wiped up next time they are generated. You should put back disabling this cop in rubocop.yml.

Reply:
Okay I didn't realize these were auto-generated.  If the rule exception were put back into`.rubocop.yml` then it would allow method naming violations in other areas of the code.  Do we want that?

Obviously as the maintainer of the project it's your call on what Rubocop rules are included. 
 My take is if we want the code to be more readable and performant, we should try to stick to a nearly empty `.rubocop.yml`.

Reply:
I agree in principle but also at some point there's signal vs. noise... anyway that's a philosophical debate.

You could modify the templates or exclude generated files as a solution if you want that. But either way modifying generated files isn't going to work.